Tenduke.Client.AspNetCore/Tenduke.Client.AspNetCore.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageIconUrl>https://raw.githubusercontent.com/10Duke/10duke-dotnet-client/master/10duke.ico</PackageIconUrl>
     <PackageIcon>10duke-192x192.png</PackageIcon>
     <RepositoryUrl>https://github.com/10Duke/10duke-dotnet-client.git</RepositoryUrl>
-    <PackageReleaseNotes>Update dependencies</PackageReleaseNotes>
+    <PackageReleaseNotes>https://github.com/10Duke/10duke-dotnet-client/releases</PackageReleaseNotes>

```

Comment:
Use the releases page as the release notes - this means that there can be a markdown document added to the releases in github detailing the changes, rather than documenting the changes embedded in this XML file.